easybuild/easyconfigs/l/libplinkio/libplinkio-0.9.8-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,33 @@
+# Contribution by
+# DeepThought, Flinders University
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'CMakeMake'
+
+name = 'libplinkio'
+version = '0.9.8'
+
+homepage = "https://github.com/mfranberg/libplinkio"
+description = """A small C and Python library for reading PLINK genotype files."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'cstd': 'gnu17', 'noopt': True, 'extra_cflags': '-w'}
```

Comment:
@manifestoso Is there a specific reason why you're enabling `noopt` here?

That results in compiling the source code with `-O0`, which probably results in terrible performance...

Reply:
Just removing `noopt` seems to work fine for the installation...

```python
toolchainopts = {'cstd': 'gnu17', 'extra_cflags': '-w'}